src/coreclr/vm/genanalysis.cpp

```diff
@@ -33,13 +34,20 @@ bool gcGenAnalysisDump = false;
         {
             match = false;
         }
-        if (match && !CLRConfig::IsConfigOptionSpecified(W("GCGenAnalysisBytes")))
+        if (match && !CLRConfig::IsConfigOptionSpecified(W("GCGenAnalysisBytes")) &&
```

Comment:
>GCGenAnalysisBytes [](http://example.com/codeflow?start=60&length=18)

I just noticed - why isn't this INTERNAL_GCGenAnalysisBytes? same with GCGenAnalysisGen.

Also these shouldn't be INTERNAL since they are specifically external facing configs...